Formula/joplin-terminal.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class JoplinTerminal < Formula
+  desc "Note taking and to-do application with synchronization capabilities"
+  homepage "https://joplinapp.org/"
+  url "https://registry.npmjs.org/joplin/-/joplin-2.2.2.tgz"
+  sha256 "15b932b51632dbcf8dfa0c39541af49b7e072b1a7c204e304d0fbdea6f1d787a"
+  license "MIT"
+
+  depends_on "node"
```

Comment:
Can we use Homebrew `vips` as a dependency here? It seems to still be needed, and the formula previously had a `vips` dependency.

Reply:
@carlocab I've added `vips` as a dependency as requested. However, I can install this on my macOS (intel chip) machine without vips being a dependency (and vips is not installed by homebrew on my system). Also, I believe the tests on mac intel hardware all pass.

It seems that adding this dependency changes the errors that block both the `11-arm64` and `ubuntu-latest` tests.

## 11-arm64

```
==> Installing 'bundler' gem
joplin-cli:
  * Binaries built for a non-native architecture were installed into joplin-cli's prefix.
    The offending files are:
      /opt/homebrew/Cellar/joplin-cli/2.2.2/libexec/lib/node_modules/joplin/node_modules/node-notifier/vendor/mac.noindex/terminal-notifier.app/Contents/MacOS/terminal-notifier	(x86_64)
Error: 1 problem in 1 formula detected
```

This seems to be because `node-notifier` doesn't work on apple's m1 chips:

https://github.com/mikaelbr/node-notifier/issues/361

I'd be very interested to hear from anyone who has an m1 chip if `yarn add joplin` or `npm install joplin` complete successfully. My suspicion is that joplin's terminal app simply doesn't work on m1 silicon due to a variety of issues with dependencies.

## ubuntu-latest

```
Error: The following formulae cannot be installed from bottles and must be
built from source.
  qt, poppler and vips
Already downloaded: /home/linuxbrew/.cache/Homebrew/downloads/da59d3b87c9ca458f5999e38c39a44b7226af89ff8f6634062a0c37eebe706a1--joplin-2.2.2.tgz
==> Verifying checksum for 'da59d3b87c9ca458f5999e38c39a44b7226af89ff8f6634062a0c37eebe706a1--joplin-2.2.2.tgz'

==> brew audit joplin-cli --online --new-formula
Error: install failed
==> FAILED joplin-cli
```

I suspect that this issue is caused by the addition of vips as a dependency, which I don't think is actually required. I suspect the original keytar issue was probably easier to resolve than this one.

Reply:
I've removed `vips` as a dependency again. I think I captured the incorrect error for the `ubuntu-latest` test without vips as a dependency.